src/Features/Core/Portable/ExtractMethod/MethodExtractor.cs

```diff
@@ -148,13 +216,15 @@ private ImmutableArray<AbstractFormattingRule> GetFormattingRules(Document docum
 
             status = result.Item2;
 
-            var checkedStatus = await CheckTypeAsync(document.Document, context, context.GetLocation(), analyzeResult.ReturnType, cancellationToken).ConfigureAwait(false);
+            var checkedStatus = CheckType(semanticModel, context, context.GetLocation(), analyzeResult.ReturnType);
             return checkedStatus.With(status);
         }
 
-        private async Task<Tuple<bool, OperationStatus>> TryCheckVariableTypeAsync(
-            SemanticDocument document, SyntaxNode contextNode, IEnumerable<VariableInfo> variables,
-            OperationStatus status, CancellationToken cancellationToken)
+        private Tuple<bool, OperationStatus> TryCheckVariableType(
```

Comment:
>Tuple<bool, OperationStatus> [](http://example.com/codeflow?start=16&length=28)

Why not just return an OperationStatus here? It looks like the bool returned is just whether it failed

Reply:
moved to mop up pr.